src/WhitespacesFixerConfig.php

```diff
@@ -25,8 +25,8 @@ final class WhitespacesFixerConfig
 
     public function __construct(string $indent = '    ', string $lineEnding = "\n")
     {
-        if (!\in_array($indent, ['  ', '    ', "\t"], true)) {
-            throw new \InvalidArgumentException('Invalid "indent" param, expected tab or two or four spaces.');
+        if (!Preg::match('/^(?: +|\t)$/', $indent)) {
```

Comment:
_IF_ we would go forward direction to approve the PR, we need a new tests to prove it works:
- each fixer that uses WhitespacesFixerConfig
- integration tests (at least mimic `tests/Fixtures/Integration/set/@PSR12_whitespaces.test*.php`)

(i suggest to not invest into it till aligned into direction)